### PR TITLE
feat: Partial arm64 support

### PR DESCRIPTION
**Pull request by DRK3**
Updated scripts to add support for running on arm64. Some issues still remain - these will be addressed in another commit.

Signed-off-by: Derek Trider <Derek.Trider@securekey.com>

### CODE REVIEW

File: k8s/cms/kustomize/cms/overlays/common/oathkeeper/process_template.sh [hunk 1]
```diff
@@ -8,9 +8,11 @@
 echo "Adding curl and jq"
 apk --no-cache add curl jq
 
+ARCH=$( uname -m | sed 's/x86_64/amd64/')
```
Comment (talwinder50) [inline k8s/cms/kustomize/cms/overlays/common/oathkeeper/process_template.sh:11]: Need to add this to have compatibility with intel machine as well. 

UNAME=$( uname -m)
case $UNAME in
   x86_64)
     ARCH="amd64"
     ;;
   aarch64)
     ARCH="arm64"
     ;;
   *)
     ARCH="amd64"
     ;;
esac

File: k8s/jobs/kustomize/jobs/overlays/common/issuer/register-tenant.sh [hunk 2]
```diff
@@ -7,9 +7,11 @@
 echo "Adding curl and jq"
 apk --no-cache add curl jq
 
+ARCH=$( uname -m | sed 's/x86_64/amd64/')
```
Comment (talwinder50) [inline k8s/jobs/kustomize/jobs/overlays/common/issuer/register-tenant.sh:10]: Need to add this here

UNAME=$( uname -m)
case $UNAME in
x86_64)
ARCH="amd64"
;;
aarch64)
ARCH="arm64"
;;
*)
ARCH="amd64"
;;
esac

File: k8s/scripts/minikube_setup.sh [hunk 3]
```diff
@@ -14,14 +14,26 @@ set -e
 : ${KUBERNETES_VERSION:=v1.21.2}
 
 OS=$( uname -s | tr '[:upper:]' '[:lower:]' )
+ARCH=$( uname -m | sed 's/x86_64/amd64/')
```
Comment (talwinder50) [inline k8s/scripts/minikube_setup.sh:17]: here as well

File: k8s/cms/kustomize/cms/overlays/common/oathkeeper/process_template.sh (hunk 1 above)
Comment (DRK3) [reply k8s/cms/kustomize/cms/overlays/common/oathkeeper/process_template.sh]: Done. Note that I removed the `*)` case as it causes `arm64` to get incorrectly converted to `amd64`.

File: k8s/jobs/kustomize/jobs/overlays/common/issuer/register-tenant.sh (hunk 2 above)
Comment (DRK3) [reply k8s/jobs/kustomize/jobs/overlays/common/issuer/register-tenant.sh]: Done. Note that I removed the `*)` case as it causes `arm64` to get incorrectly converted to `amd64`.

File: k8s/scripts/minikube_setup.sh (hunk 3 above)
Comment (DRK3) [reply k8s/scripts/minikube_setup.sh]: Done - but note that this doesn't run in a container, so on Mac there's no need for an `aarch64` to `arm64` name conversion...but...for completeness and best support it's probably a good idea to have it!

**Review by talwinder50 — APPROVED** [review]

**Review by birtony — APPROVED** [review]

**Pull request merged**